tests/test_generators/test_name_compatibility.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+from linkml.generators.common.name_compatibility import NameCompatibility
+
+
+def test_equality_graphql():
+    name_compatibility = NameCompatibility(profile="graphql")
+    print(name_compatibility.profile)
```

Comment:
no print statements plz :)

(also if you're not running the tests with pytest, where print statements wouldn't be shown, you can run test modules like `pytest tests/test_generators/test_name_compatibility.py` or individual tests like `pytest -k "test_equality_graphql"`, and to inspect outputs at any given point use `pdb.set_trace()` (or warn if print is what is really desired) - tests basically should never be run in a circumstance where printing would be visible (ie. should always be run with pytest) because if they are just run as modules we don't get all the pytest behavior (so we can get false positive or negative failures) and discourage ourselves from doing things like using fixtures

Reply:
Just another late-night error. I only had it for debugging purposes.

That print worked exactly as expected: it was only shown when I was getting an unexpected error.

Removing it.